src/elements/ticket-element.html

```diff
@@ -155,7 +155,7 @@
         ],
 
         properties: {
-          type: {
+          name: {
```

Comment:
Ok. I didn't see type and type. I saw "name" and type. My bad.

    "elements": [
      {
        "name": "student-ticket",
        "price": 500,
        "currency": "₴",
        "starts": "student-ticket-starts",
        "ends": "student-ticket-ends",
        "info": "student-ticket-info",
        "soldOut": false
      }
    ]